lifecycle-operator/controllers/common/helperfunctions.go

```diff
@@ -15,6 +15,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const KeptnNamespace = "keptn"
```

Comment:
please, move it back into the test file since in production code we cannot assume the namespace. Furthermore, based on our docs the default should be `keptn-lifecycle-toolkit-system`

Reply:
moved to `test_utils.go`